namesys/ipns_publisher.go

```diff
@@ -113,7 +111,7 @@ func (p *IpnsPublisher) ListPublished(ctx context.Context) (map[peer.ID]*ipns.Re
 //
 // If `checkRouting` is true and we have no existing record, this method will
 // check the routing system for any existing records.
-func (p *IpnsPublisher) GetPublished(ctx context.Context, id peer.ID, checkRouting bool) (*ipns.Record, error) {
+func (p *IPNSPublisher) GetPublished(ctx context.Context, id peer.ID, checkRouting bool) (*ipns.Record, error) {
```

Comment:
Similarly to the comment to `ListPublished`, I'm not sure why this was exported. It's only used internally.

Reply:
It feels useful?  Allows user of the `boxo/namesys` library to do  local lookup without hitting remote routing.
I'd say we keep it but replace `peer.ID` with `ipns.Name` (since we make breaking changes anyway, remove all `peer.ID` from public apis in `boxo/namesys`?